conf/config.level0.neon

```diff
@@ -115,6 +117,7 @@ rules:
 	- PHPStan\Rules\Types\InvalidTypesInUnionRule
 	- PHPStan\Rules\Variables\UnsetRule
 	- PHPStan\Rules\Whitespace\FileWhitespaceRule
+	- PHPStan\Rules\Keywords\RequireFileExistsRule
```

Comment:
This can't be in the rules section, it'd always be registered, regardless of the conditionalTags.

Needs to be in services.

Reply:
Fixed in https://github.com/phpstan/phpstan-src/pull/3294/commits/62d47af944d4db90f4c56f1bc7dfe0e04067c843